### PR TITLE
[Tests-Only]Add acceptance tests for breadcrumbs with double quotes

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR adds acceptance tests for breadcrumbs with double quotes and edits selector accordingly

## Related Issue
- Fixes https://github.com/owncloud/phoenix/issues/3734

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by jasson99** [comment]
This still doesnot work for single quote

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -1089,6 +1093,19 @@ When('the user moves file/folder {string} into folder {string} using the webUI',
...
+) {
+  return client.page.FilesPageElement.filesList().attemptToMoveResource(resource, target)
+})
+
+Then('it should not be possible to copy/move into folder {string} using the webUI', function(
+  target
+) {
+  return client.page.FilesPageElement.filesList().navigationNotAllowed(target)
+})
+
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/filesContext.js:1108]: changes look unrelated

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 2]
```diff
@@ -1107,6 +1124,13 @@ When('the user copies file/folder {string} into folder {string} using the webUI'
   return client.page.FilesPageElement.filesList().copyResource(resource, target)
 })
 
+When('the user tries to copy file/folder {string} into folder {string} using the webUI', function(
+  resource,
+  target
+) {
+  return client.page.FilesPageElement.filesList().attemptToCopyResource(resource, target)
+})
+
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/filesContext.js:1133]: changes look unrelated

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/stepDefinitions/filesContext.js]: resolved

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/stepDefinitions/filesContext.js]: resolved
